### PR TITLE
Fix transaction type module

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR is about the transaction type fixed record in the ui grid

### CODE REVIEW

**Comment by mbayopanda** [comment]
@jniles could you review this

File: client/src/partials/admin/transaction_type/transaction_type.js [hunk 1]
```diff
@@ -20,10 +20,14 @@ function TransactionTypeController(TransactionType, TransactionTypeStore, Notify
 
   // edit button template
   var editTemplate = '<div class="ui-grid-cell-contents">' +
-    '<a title="{{ \'FORM.LABELS.EDIT\' | translate }}" href="" ' +
+    '<a href="" title="{{ \'FORM.LABELS.EDIT\' | translate }}" ' +
     'ng-click="grid.appScope.editType(row.entity)" ' +
+    'uib-popover="{{grid.appScope.notAllowed(row.entity.fixed) | translate }}" ' +
```
Comment (jniles) [inline client/src/partials/admin/transaction_type/transaction_type.js:25]: Does this make an empty `uib-popover` if `notAllowed()`?

**Comment by jniles** [comment]
I don't have time to pull and test this - if the `uib-popover` doesn't exist for `notAllowed()` rows, then I'm happy to merge this.

File: client/src/partials/admin/transaction_type/transaction_type.js (hunk 1 above)
Comment (mbayopanda) [reply client/src/partials/admin/transaction_type/transaction_type.js:25]: transaction-type

If the row is not fixed, the popover doesn't appear

**Comment by mbayopanda** [comment]
@jniles could you review this PR

**Comment by jniles** [comment]
LGTM. :+1:

**Pull request merged**